src/components/shared/Modal.css

```diff
@@ -0,0 +1,47 @@
+.modal-wrapper {
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  z-index: 9;
+  top: 0;
+  left: 0;
+}
+
+.modal-wrapper.entering,
+.modal-wrapper.exited {
+  pointer-events: none;
+}
+
+.modal-wrapper.entered,
+.modal-wrapper.exiting {
+  pointer-events: auto;
+}
```

Comment:
we should avoid having a div overlaying the window at all times.

another strategy might be to use z-index: -1.